resources/grafana/generated/dashboards/rhacs-central.json

```diff
@@ -0,0 +1,5229 @@
+{
```

Comment:
I think the file structure is a bit unfortunate as the `.json` here is the source of truth for the dashbord `.yaml`. So I would rather put the latter into a `/generated` folder and keep the `.json` more top level.